pkgs/applications/networking/znc/default.nix

```diff
@@ -3,6 +3,7 @@
 , withPython ? false, python3
 , withTcl ? false, tcl
 , withCyrus ? true, cyrus_sasl
+, withUnicode ? false, icu
```

Comment:
Should probably be enabled by default.

Check the closure size difference with that.

Reply:
Updated